part2/stages/Functions/install-main

```diff
@@ -1033,5 +1033,10 @@ own_tpm() {
 
     generate_policy "${own_key}"
 
+    if [ $? -ne 0 ]; then
+        echo "error generating tboot policy" >&2
```

Comment:
Is there any reason to log the failure here? From what I can tell, everywhere in ```generate_policy``` when an error is encountered it messages specifically why it failed before returning the non-zero return value. IMHO, if there is a failure in ```generate_policy``` that is not does not have a corresponding log message, it should be fixed there instead of a course grain message at this level.

Reply:
I added this more for the `return 1` rather than the log. This function (`own_tpm`) will return 0 whether or not generate_policy fails, and that was causing some headaches while debugging.

Reply:
Apologies, as I should have been more clear. I am good with the return of 1, I wasn't sure if the additional log message was because a failure was missed. If it was then that should be fixed in the generate_policy function. If it wasn't, then this log message is noise which is something we should be looking to reduce.

Reply:
There was no reason for the log itself, no. I will remove that and let `generate_policy` handle the logging, which seems to be pretty thorough.